_episodes/format.md

```diff
@@ -88,6 +88,8 @@ Here are some recommended formats for different types:
 > Data Management Plan (DMP) covers how data can be stored, described and reused.
 {: .callout}
 
+_AK: For example, [DMPonline](https://www.dcc.ac.uk/dmponline) or [DSW](https://ds-wizard.org/)
+
```

Comment:
Is this meant as a comment? If so, please remove this statement and add the links where you think they would be most appropriate.

```suggestion
```